dbump_pgx/pgx.go

```diff
@@ -27,8 +27,8 @@ func NewMigrator(conn *pgx.Conn) *Migrator {
 // Init migrator.
 func (pg *Migrator) Init(ctx context.Context) error {
 	query := `CREATE TABLE IF NOT EXISTS _dbump_schema_version (
-	version    BIGINT INT NOT NULL PRIMARY KEY,
-	created_at TIMESTAMP NOT NULL
+	version    BIGINT                   NOT NULL PRIMARY KEY,
+	created_at TIMESTAMP WITH TIME ZONE NOT NULL
```

Comment:
Why so?

Reply:
TIMESTAMP without time zone it is a bug in most cases.
Just a habit :)

https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_timestamp_.28without_time_zone.29